src/routes/conceptRoutes.js

```diff
@@ -26,7 +26,8 @@ module.exports = function (app) {
     .get(requestMiddleware.createAndValidateRequestBody, domainService.getConceptByIdAPI)
 
   app.route(BASE_URL_V1 + '/domains/:domainId/:objectType/search')
-    .post(requestMiddleware.createAndValidateRequestBody, domainService.searchObjectTypeAPI)
+    .post(requestMiddleware.createAndValidateRequestBody, requestMiddleware.addChannelFilters,
```

Comment:
@rajeevsathish should this be added to other aspects of content fetching? are channel filters not required for concepts and domain ids?